esmvalcore/_config/extra_facets/ipslcm-mappings.yml

```diff
@@ -0,0 +1,239 @@
+  OcePtr: &oceptr {group: ptrc_T, dir: BGC, <<: *gene}
+  IceVars: &icevars {group: icemod, dir: ICE, <<: *gene}
+
+
+IPSL-CM6:
+  # ========================================================================================
+  Amon:
+    # ========================================================================================
+    # ATM 3D Variables
+    ta: {ipsl_varname: ta, <<: *atm3dvars}
+    ua: {ipsl_varname: ua, <<: *atm3dvars}
+    va: {ipsl_varname: va, <<: *atm3dvars}
+    zg: {ipsl_varname: zg, <<: *atm3dvars}
+    hus: {ipsl_varname: hus, <<: *atm3dvars}
+    hur: {ipsl_varname: hur, <<: *atm3dvars}
+
+    # ATM general variables
+    pr: {ipsl_varname: precip, <<: *atmvars}
+    psl: {ipsl_varname: slp, <<: *atmvars}
+    tas: {ipsl_varname: t2m, <<: *atmvars}
+    ts: {ipsl_varname: tsol, <<: *atmvars}
+    huss: {ipsl_varname: q2m, <<: *atmvars}
+    uas: {ipsl_varname: u10m, <<: *atmvars}
+    "vas": {ipsl_varname: v10m, <<: *atmvars}
+    sfcWind: {ipsl_varname: wind10m, <<: *atmvars}
+    hurs: {ipsl_varname: rh2m, <<: *atmvars}
+
+    # ATM general variables
+    precip: {ipsl_varname: precip, <<: *atmvars}
+    prw: {ipsl_varname: prw, <<: *atmvars}
+    slp: {ipsl_varname: slp, <<: *atmvars}
+    t2m: {ipsl_varname: t2m, <<: *atmvars}
+    q2m: {ipsl_varname: q2m, <<: *atmvars}
+    u10m: {ipsl_varname: u10m, <<: *atmvars}
+    v10m: {ipsl_varname: v10m, <<: *atmvars}
+    wind10m: {ipsl_varname: wind10m, <<: *atmvars}
+
+    #  -> Turbulent fluxes
+    taux: {ipsl_varname: taux, <<: *atmvars}
+    tauy: {ipsl_varname: tauy, <<: *atmvars}
+
+    #  -> Radiative down at TOA
+    SWdnTOA: {ipsl_varname: SWdnTOA, <<: *atmvars}
+
+    #  -> Radiative down at TOA
+    topl: {ipsl_varname: topl, <<: *atmvars}
+    SWupTOA: {ipsl_varname: SWupTOA, <<: *atmvars}
+    topl0: {ipsl_varname: topl0, <<: *atmvars}
+    SWupTOAclr: {ipsl_varname: SWupTOAclr, <<: *atmvars}
+
+    #  -> Radiative up at Surface
+    LWupSFC: {ipsl_varname: LWupSFC, <<: *atmvars}
+    SWupSFC: {ipsl_varname: SWupSFC, <<: *atmvars}
+    SWupSFcclr: {ipsl_varname: SWupSFcclr, <<: *atmvars}
+    LWupSFcclr: {ipsl_varname: LWupSFcclr, <<: *atmvars}
+
+    #  -> Radiative down at Surface
+    LWdnSFC: {ipsl_varname: LWdnSFC, <<: *atmvars}
+    SWdnSFC: {ipsl_varname: SWdnSFC, <<: *atmvars}
+    LWdnSFcclr: {ipsl_varname: LWdnSFcclr, <<: *atmvars}
+    SWdnSFcclr: {ipsl_varname: SWdnSFcclr, <<: *atmvars}
+
+    #  -> Turbulent fluxes
+    hfls: {ipsl_varname: flat, <<: *atmvars, positive: down}
+    hfss: {ipsl_varname: sens, <<: *atmvars, positive: down}
+    tauu: {ipsl_varname: taux, <<: *atmvars}
+    tauv: {ipsl_varname: tauy, <<: *atmvars}
+
+    #  -> Clouds
+    clt: {ipsl_varname: cldt, <<: *atmvars}
+
+    #  -> Radiative up at TOA
+    rlut: {ipsl_varname: topl, <<: *atmvars}
+    rsut: {ipsl_varname: SWupTOA, <<: *atmvars}
+    rlutcs: {ipsl_varname: topl0, <<: *atmvars}
+    rsutcs: {ipsl_varname: SWupTOAclr, <<: *atmvars}
```

Comment:
This seems to be a repetition of some lines above with new names. Is this section split into "IPSL style names" first and then "CMIP style names" after that? If so, I think we should consider to leave out the IPSL style names since they might lead to confusion, or otherwise add comments for the respective sub-sections.

Reply:
You're right. I organized a subsection commented as "ATM variables processed using their own IPSL short_name"